### ISSUE TITLE
Batchnorm

### ISSUE THREAD

**Issue by jamaliki** [e3nn/e3nn-jax#2]
I implemented a batch norm based on the e3nn repository. I can paste the code here or if you give me access to make a branch, I can create a PR. @mariogeiger
